book_config.py

```diff
@@ -32,54 +31,36 @@ def __init__(self, db, ids, parent):
             self._proxy = True
             self._http_address = ':'.join(http_proxy.split(':')[:-1])
```

Comment:
Just spotted a problem here - http_proxy is looking up the http proxy, not the https proxy.  I assume they can be different as we are looking http up explicitly.